manifest.md

```diff
@@ -135,7 +135,7 @@ Unlike the [Manifest List](#manifest-list), which contains information about a s
 - **`config`** *[descriptor](descriptor.md)*
 
     This REQUIRED property references a configuration object for a container, by digest.
-    The referenced configuration object is a JSON blob that the runtime uses to set up the container, see [Image JSON Description](serialization.md#image-json-description).
+    The referenced configuration object is a JSON blob that the runtime uses to set up the container, see [Image JSON Description](config.md#image-json-description).
```

Comment:
Now that the files are split, we may want to drop the fragment here.  If `config.md` is so big that you want to link to a subsection as the spec for the format, we should probably trim `config.md` ;).